### ISSUE TITLE
2-stage training

### ISSUE THREAD

**Issue by Yutasq** [RomeroBarata/hierarchical_action_prediction#2]
Hi Romero,

Thanks for sharing the code. In the paper you mentioned 2-stage training, does it mean we should run train.py two times, and in the second training loading the pre-trained model while changing parameters of the segment loss and pattern?

Thanks and looking forward to your reply. 

Best,
Yutasq

**Comment by RomeroBarata** [comment]
Hi Yutasq,

I believe you are referring to this other codebase https://github.com/RomeroBarata/human_object_interaction, right? In that case, yes, you have to run `train.py` twice as you described.

Kind regards,
Romero
